orb-ui/src/engine/animations/arc_dash.rs

```diff
@@ -63,6 +63,10 @@ impl<const N: usize> ArcDash<N> {
 impl<const N: usize> Animation for ArcDash<N> {
     type Frame = RingFrame<N>;
 
+    fn name(&self) -> &'static str {
```

Comment:
It is possible to make this a `const NAME: &str;` in the trait instead of a method? Asking if possible because im not sure if that will work with trait objects.